docs/statistics/bivariate/Correlation.md

```diff
@@ -16,7 +18,23 @@ In the previous univariate parts, the metrics focused only on a single variable.
 
 The name suggests that it is a type of 'variance,' as \( \text{cov}(X, X) = \sigma^2(X) \).
 
-??? example
+
+```py
+import plotly.express as px
+df = px.data.tips()
+
+Covariance = df[['total_bill', 'tip']].cov(ddof=0)
```

Comment:
Just a minor suggestion: Would you prefer to do the calculation with `pandas` Series, just like you did in the next couple of examples?
```suggestion
Covariance = df['total_bill'].cov(df['tip'], ddof=0)
print(f"Covariance: {Covariance}")
```